hawc/apps/summary/models.py

```diff
@@ -417,6 +418,22 @@ def get_prisma_data(cls, assessment: Assessment) -> str:
             }
         )
 
+    @classmethod
+    def get_data_from_config(cls, config: schemas.VisualConfig):
```

Comment:
also, we'll always know the assessment ID, external to the config. I'd take that as an input argument instead of getting it from the config; if we relied on config a bad actor could submit a different assessment in their config from a visual assessment or assessment.